frontend/packages/dev-console/src/components/pipelineruns/PipelineRunLogs.tsx

```diff
@@ -127,4 +146,17 @@ class PipelineRunLogs extends React.Component<PipelineRunLogsProps, PipelineRunL
   }
 }
 
+type PipelineRunLogsWithActiveTaskProps = {
+  obj: PipelineRun;
+  params?: any;
```

Comment:
fixed in https://github.com/openshift/console/pull/2695/commits/76d8c8385d320dcb938e94579f1e7a4018d56c70
Used `RouteComponentProps` because it also includes history and location.